src/Microsoft.DocAsCode.Build.Engine/PostProcessors/IncrementalPostProcessorHost.cs

```diff
@@ -29,8 +30,16 @@ public IncrementalPostProcessorHost(IncrementalPostProcessorsContext increContex
 
             _increContext = increContext;
             _postProcessorName = postProcessorName;
+            SourceFileInfos = sourceFileInfos;
```

Comment:
just found that  the mapping is between outputfile and bookmarks, namely what is needed is nonincrementalitems rather than sourceFileInfo..